Add gedit theme

richin13 opened the pull request:
This is the same PR as [zenorocha/dracula-theme](https://github.com/zenorocha/dracula-theme)'s [#185](https://github.com/zenorocha/dracula-theme/pull/185)

The installations instructions were added to the README file and I updated all the values as corresponded. If something is incorrect please let me know.

richin13 committed:
Add Gedit theme

I added the Gedit's dracula theme along with the change made by
@qolop that consisted in changing the cursor's color to white
because before that It was in the same color as the background
which made it invisible.

richin13 committed:
Fix markdown format issues

nesl247 commented on README.md:
```diff
@@ -1,21 +1,36 @@
+> A dark theme for [Gedit](https://wiki.gnome.org/Apps/Gedit).
 
-![Screenshot](https://draculatheme.com/assets/img/screenshots/sublime.png)
+![Screenshot](http://i.imgur.com/paelbpq.png)
```
Please keep the URL as it was but replace `sublime` with `gedit`. When the PR is merged in that we talked about for the screenshot the url will become valid.

nesl247 commented:
FYI once we've gotten this PR completed and merged in I will add you to the team for Gedit.

richin13 committed:
Update README.md

richin13 commented:
Great

nesl247 commented:
There's definitely some differences compared to the same code in Atom. Here's a rough screenshot showing them. Is there any way to get them closer? I just created another issue #192 for us to have an official list in the future.

![screen shot 2016-05-30 at 6 17 22 pm](https://cloud.githubusercontent.com/assets/1037526/15659100/1cee658e-2693-11e6-80bb-c67cca377233.png)

richin13 commented:
Sure no problem.. I'll get to work later

nesl247 commented:
Hey @richin13, just curious to see how things are going? If you can add `screenshot.png` to this PR, and link to it via `+![Screenshot](./screenshot.png)` instead, I'll go ahead and merge this in, and then you can work on getting the theme a bit cleaned up to match Atom after. We're going to be working on standardizing the way themes are built for editors soon, so that will help with that.

richin13 commented:
Hi @nesl247, I was a little busy with college's assignments but I will get to work this weekend. I will upload the screenshot as soon as is ready.

nesl247 commented:
No problem at all, just wanted to check in and see. I just got done with a move, so I was busy too.

richin13 committed:
Change the theme's look

Also the code was properly ordered and formatted.
The colours are now variables that can be reused.

richin13 committed:
Merge branch 'master' of github.com:richin13/gedit

richin13 commented:
Hi @nesl247, I have been working in Gedit's dracula theme and got actually really close to Atom's version, however, there are still some differences, mainly because the way they interpret the Python Language Specification as they group the keywords and the builtins preventing individual styling  of each one.

For example, if I style the `class` and `def` keywords to be cyan color (as in Atom's version) also gets styled the the `if`, `while`, `for` and a long etc of keywords that are supposed to be pink.

The thing is that in order to fix this problem, a couple of installation files (gedit's) should be overwritten and this would put a considerable amount of complexity in the end-user installation process (not really a good idea). And, although I haven't try other languages yet,  that would be only for Python, the same would apply to other languages.

My question here is, should it look exactly like in Atom? I have been using dracula on SublimeText and they vary between each other and right now, the screenshot attached looks more like Sublime's version than that Atom's. (There are some differences, though)

![Imgur](http://i.imgur.com/bPNELMN.png)

If you consider that the current version it's an acceptable implementation of Dracula Theme I would only need to work in the way other languages looks and the theme would be ready. In the case you consider is not acceptable I would have not problem but I would need some more time to dig the documentation and find something that I could be missing right now.

Greetings

nesl247 commented:
Hey @richin13,

Don't worry about making it exact. Every editor or IDE seems to group things differently, so we can only do the best with what we have access to.

Once you've taken care of the screenshot issue I mentioned above, I'll go ahead and merge this in. We can improve the theme as time goes on, but at least people will be able to use it as it develops :).

Thanks again!

richin13 committed:
Add theme screenshot

richin13 committed:
Update README.md

Add proper screenshot path.

richin13 commented:
Great! It's done
